plugins/UsersManager/lang/en.json

```diff
@@ -8,7 +8,8 @@
         "AnonymousUserHasViewAccess2": "Your analytics reports and your visitors information are publicly viewable.",
         "ApplyToAllWebsites": "Apply to all websites",
         "ChangeAllConfirm": "Are you sure you want to give '%s' access to all the websites?",
-        "ChangePasswordConfirm": "Changing the password will also change the user's token_auth. Do you really want to continue?",
+        "ChangePasswordConfirm": "Changing the password will also change the user's old API authentication token which may be used to access the Piwik APIs. This token will be re-generated only once. This is a security measure to keep your Piwik data safe. Do you really want to continue?",
+        "ChangePasswordConfirmSelf": "Changing the password will also change the your old API authentication token which may be used to access the Piwik APIs. This token will be re-generated only once. This is a security measure to keep your Piwik data safe. Proceeding may require you to log in again. Do you really want to continue?",
```

Comment:
"change the your old API authentication"

=> "change your old API authentication" :)